core/arch/riscv64/riscv64.asm

```diff
@@ -208,16 +208,51 @@ ADDRTAKEN_LABEL(safe_read_asm_recover:)
  */
         DECLARE_EXPORTED_FUNC(dr_try_start)
 GLOBAL_LABEL(dr_try_start:)
-        addi      ARG1, ARG1, TRY_CXT_SETJMP_OFFS
+        addi     ARG1, ARG1, TRY_CXT_SETJMP_OFFS
         j        GLOBAL_REF(dr_setjmp)
         END_FUNC(dr_try_start)
 
-/*
+/* We save only callee-saved regites: SP, x8/fp, x9, x18-x27, f8-9, f18-27:
```

Comment:
Do you mean s/regites/regs/?

Reply:
Done.

Reply:
Fixed https://github.com/DynamoRIO/dynamorio/pull/6071/commits/3859e2075255495463f4d09652e54771306f0a75